recipes/pbzip2/build.sh

```diff
@@ -0,0 +1,2 @@
+make PREFIX=$PREFIX CXXFLAGS="-I$PREFIX/include -L$PREFIX/lib $CXXFLAGS"
```

Comment:
The `-L` doesn't need to be in `CXXFLAGS`. If you want to add the lib folder to the linking path you should add it to `LDFLAGS`.